src/ontology/mondo-edit.obo

```diff
@@ -437832,6 +437838,8 @@ xref: ICD10CM:Q50-Q56 {source="https://github.com/monarch-initiative/mondo/issue
 xref: ICD10CM:Q60-Q64 {source="https://github.com/monarch-initiative/mondo/issues/4536", source="MONDO:equivalentTo", source="https://orcid.org/0000-0001-5208-3432", source="https://orcid.org/0000-0002-4142-7153"}
 xref: Orphanet:83001 {source="MONDO:equivalentTo"}
 is_a: MONDO:0019755 {source="Orphanet:83001"} ! developmental defect during embryogenesis
+intersection_of: MONDO:0019755 ! developmental defect during embryogenesis
+intersection_of: has_material_basis_in_germline_mutation_in http://identifiers.org/hgnc/15979 ! TP63
 
```

Comment:
@yshwetar This logical definition was added to the wrong term. I will move at it to the correct one.

Reply:
Actually, I don't think that this logical definition is correct. (see comment on the definition)

Reply:
Whoops! Yea that happened a few times and I thought I caught them all. Thank you!